patches/api/0005-Adventure.patch

```diff
@@ -62,6 +62,9 @@ index 5fccd83117c2c153951d102756acfd4862b8c51b..acf12dfbecb9360c0921f70ed31650fe
 +        "https://jd.advntr.dev/text-serializer-gson/$adventureVersion/",
 +        "https://jd.advntr.dev/text-serializer-legacy/$adventureVersion/",
 +        "https://jd.advntr.dev/text-serializer-plain/$adventureVersion/",
++        "https://jd.advntr.dev/text-logger-slf4j/$adventureVersion/",
++        "https://javadoc.io/doc/org.slf4j/slf4j-api/1.8.0-beta4",
++        "https://javadoc.io/doc/org.apache.logging.log4j/log4j-api/2.17.1/"
```

Comment:
I'm not sure these should go here, but in an earlier patch like build system changes or smth. Just the adventure-specific one should be here I think.